Android/Yash Raj Gupta/Beginner/Contacts/app/src/main/java/com/example/contacts/CustomAdapter.java

```diff
@@ -0,0 +1,48 @@
+package com.example.contacts;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+public class CustomAdapter extends BaseAdapter {
+    Context context;
+    String actor[];
+    int image[];
+    LayoutInflater inflter;
+
+    public CustomAdapter(Context applicationContext, String[] actor, int[] image) {
+        this.context = context;
+        this.actor =  actor;
+        this.image = image;
+        inflter = (LayoutInflater.from(applicationContext));
+    }
+
+    @Override
+    public int getCount() {
+        return actor.length;
+    }
+
+    @Override
+    public Object getItem(int i) {
+        return null;
+    }
+
+    @Override
+    public long getItemId(int i) {
+        return i;
+    }
+
+    @Override
+    public View getView(int i, View view, ViewGroup viewGroup) {
+        view = inflter.inflate(R.layout.list_view, null);
+        TextView country = (TextView) view.findViewById(R.id.textView);
+        ImageView icon = (ImageView) view.findViewById(R.id.icon);
+        country.setText("  " + actor[i]);
```

Comment:
As per the current implementation, contacts will not be displayed in alphabetical order. Please check.